shared/control/stress_memory_heavy.control

```diff
@@ -11,22 +11,21 @@ to heavily stress a machine's memory.
 '''
 
 import os
-from autotest.client import utils
 
-try:
-    from virttest.staging import utils_memory
-except ImportError:
-    from autotest.client.shared import utils_memory
+from avocado.utils import cpu as cpu_utils
+from avocado.utils import disk as disk_utils
+
+from virttest.staging import utils_memory
 
 # Assemble the parameters specially to stress memory
 
 # We will use 2 workers of each type for each CPU detected
-threads = 2 * utils.count_cpus()
+threads = 2 * cpu_utils.total_cpus_count()
```

Comment:
utils.count_cpus() in autotest gets online cpu counts, it may be different with total_cpus_count() in avocado,
please double check

Reply:
@chunfuwen Fixed